src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackSaveAction.java

```diff
@@ -0,0 +1,195 @@
+        try {
+            logic.deleteInstructorRespondant(getUserEmailForCourse(), feedbackSessionName, courseId);
+        } catch (InvalidParametersException | EntityDoesNotExistException e) {
+            log.severe("Fail to remove instructor respondant");
+        }
+    }
+
+    /**
+     * Retrieves the email of the user for the course.
+     * @return the email of the user
+     */
+    @Override
+    protected String getUserEmailForCourse() {
+        return moderatedInstructor.email;
+    }
+    
+    /**
+     * Retrieves the user's team.
+     * @return the name of the user's team
+     */
+    @Override
+    protected String getUserTeamForCourse() {
+        return Const.USER_TEAM_FOR_INSTRUCTOR;
+    }
+    
+    /**
+     * Retrieves the user's section for the course.
+     * @return the name of the user's section
+     */
+    @Override
+    protected String getUserSectionForCourse() {
+        return Const.DEFAULT_SECTION;
+    }
+
+    /**
+     * Gets data bundle for the course specified.
+     * @param userEmailForCourse the email of the user
+     * @return FeedbackSessionQuestionsBundle object
+     */
+    @Override
+    protected FeedbackSessionQuestionsBundle getDataBundle(String userEmailForCourse)
+            throws EntityDoesNotExistException {
+        return logic.getFeedbackSessionQuestionsBundleForInstructor(
+                feedbackSessionName, courseId, userEmailForCourse);
+    }
+
+    /**
+     * Sets the message to log.
+     */
+    @Override
+    protected void setStatusToAdmin() {
+        statusToAdmin = "Instructor moderated instructor session<br>" +
+                        "Instructor: " + account.email + "<br>" + 
+                        "Moderated Instructor: " + moderatedInstructor + "<br>" +
+                        "Session Name: " + feedbackSessionName + "<br>" +
+                        "Course ID: " + courseId;
+    }
+
+    /**
+     * Checks if the session is still open. However, since the instructor is moderating the session,
+     * they can moderate it anytime. Therefore, it will be true forever.
+     * @return true
+     */
+    @Override
+    protected boolean isSessionOpenForSpecificUser(FeedbackSessionAttributes session) {
+        // Feedback session closing date does not matter. Instructors can moderate at any time
+        return true; 
+    }
+
+    /**
+     * Creates the page to redirect.
+     * @return RedirectResult object
+     */
+    @Override
+    protected RedirectResult createSpecificRedirectResult() {
+        RedirectResult result = createRedirectResult(Const.ActionURIs.INSTRUCTOR_EDIT_INSTRUCTOR_FEEDBACK_PAGE);
+        
+        result.responseParams.put(Const.ParamsNames.COURSE_ID, moderatedInstructor.courseId);
+        result.responseParams.put(Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionName);
+        result.responseParams.put(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_STUDENT, moderatedInstructor.email);
```

Comment:
Creating a new constant FEEDBACK_SESSION_MODERATED_INSTRUCTOR may be more appropriate.